### PR TITLE
removed redundant lines from graphs

### PR DESCRIPTION
**Pull request by osherdp**

### CODE REVIEW

**Comment by osherdp** [comment]
@avrahamshukron you're right
Screenshot from 2020-03-30 15-30-51
Screenshot from 2020-03-30 15-30-20

**Comment by avrahamshukron** [comment]
Don't you think the threshold line adds value?

**Comment by osherdp** [comment]
Ron insists they don't matter
You wanna recheck?

**Comment by osherdp** [comment]
Sorry, you're right
Only bars in the flow should be removed

File: graphics/graphs.py [hunk 1]
```diff
@@ -68,19 +68,6 @@ def __init__(self, parent, measurements, blank):
...
-                                    [self.config.pressure_threshold.max] *
-                                    len(self.measurements.x_axis),
-                                    color=MAX_TRHLD_COLOR, linestyle=":", animated=True)
-
-        self.pressure_min_threshold_graph, = \
-            self.pressure_axis.plot(self.measurements.x_axis,
-                                    [self.config.pressure_threshold.min] *
-                                    len(self.measurements.x_axis),
-                                    color=MIN_TRHLD_COLOR, linestyle=":", animated=True)
-
```
Comment (rinslow) [inline graphics/graphs.py]: return thesse

**Comment by osherdp** [comment]
closes #83

**Pull request merged**